src/types/payout.ts

```diff
@@ -0,0 +1,22 @@
+// available tokens for payouts
+export const PAYMENT_TOKEN_PER_NETWORK: Record<string, { rpc: string; token: string; symbol: string }> = {
+  "1": {
+    rpc: "https://rpc-bot.ubq.fi/v1/mainnet",
```

Comment:
Our RPCs don't work anymore

Reply:
True, just picked this up from the previous codebase. https://rpc.gnosischain.com should do it for both then?